scim2-sdk-common/src/main/java/com/unboundid/scim2/common/GenericScimResource.java

```diff
@@ -469,6 +479,42 @@ public String toString()
     }
   }
 
+  /**
+   * Indicates whether the provided object is equal to this generic SCIM
+   * resource.
+   *
+   * @param o   The object to compare.
+   * @return    {@code true} if the provided object is equal to this generic
+   *            SCIM resource, or {@code false} if not.
+   */
+  @Override
+  public boolean equals(final Object o)
```

Comment:
I added this since I needed it in the unit tests. It felt wrong to use `assertEquals(first.getObjectNode(), second.getObjectNode())`. I also added a test for this method too.